src/locales/en.json

```diff
@@ -316,41 +307,41 @@
       "instruction": "Only your face can be in the selfie"
     },
     "server_error": {
-      "message": "Connection lost",
+      "message": "$errors.server_error.message4",
       "instruction": "Please try again"
     },
     "glare_detected": {
       "message": "Glare detected",
       "instruction": "All details should be clear and readable"
     },
     "sms_failed": {
-      "message": "Something's gone wrong",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Copy the link to your phone",
+      "instruction": "Something’s gone wrong"
     },
     "sms_overuse": {
-      "message": "Too many resend attempts",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Copy the link to your phone",
+      "instruction": "Too many failed attempts"
     },
     "lazy_loading": {
       "message": "An error occurred while loading the component"
     },
     "invalid_number": {
-      "message": "Check your mobile number is correct"
+      "message": "Check that your number is correct"
     },
     "generic_client_error": {
-      "message": "Something's gone wrong",
+      "message": "Something’s gone wrong",
```

Comment:
Why using a different character for the apostrophe? Are they equivalent?

Reply:
no idea, I can change the spreadsheet if need be

Reply:
fixed